miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/MenuController.java

```diff
@@ -226,10 +224,6 @@ public ModelAndView mainMenu(ModelMap model) {
     try {
       User user = securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName());
       model.put("title", "Home");
-      Map<String, String> checks = MisoWebUtils.checkStorageDirectories((String) servletContext.getAttribute("miso.baseDirectory"));
-      if (checks.keySet().contains("error")) {
-        model.put("error", checks.get("error"));
-      }
       ObjectMapper mapper = new ObjectMapper();
       model.put("favouriteWorkflows",
           user.getFavouriteWorkflows().stream().map(Dtos::asDto).map(dto -> mapper.valueToTree(dto)).collect(Collectors.toList()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)